.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to get an issue fixed
+title: ''
+labels: bug
+assignees: Axel-Naumann
+
+---
+
+**Describe the bug**
+A clear and concise description of what the wrong behavior is.
```

Comment:
The titles could be actual headers and the explanations could be invisible comments. The first is for ease of reading and the second so that you don't have to bother deleting the text every time you open an issue (for devs and power issue-openers).

```suggestion
## Problematic behavior**
<!-- A clear and concise description of what the wrong behavior is. -->
```

Reply:
👍 for the headers. Also remove the `**` after the header.